Core/GameInstance.cs

```diff
@@ -122,20 +122,6 @@ private void SetupCkanDirectories(bool scan = true)
                 User.RaiseMessage(Properties.Resources.GameInstanceCreatingDir, InstallHistoryDir());
                 txFileMgr.CreateDirectory(InstallHistoryDir());
             }
-
-            // Clear any temporary files we find. If the directory
-            // doesn't exist, then no sweat; FilesystemTransaction
-            // will auto-create it as needed.
-            // Create our temporary directories, or clear them if they
-            // already exist.
-            if (Directory.Exists(TempDir()))
-            {
-                var directory = new DirectoryInfo(TempDir());
-                foreach (FileInfo file in directory.GetFiles())
-                    txFileMgr.Delete(file.FullName);
-                foreach (DirectoryInfo subDirectory in directory.GetDirectories())
-                    txFileMgr.DeleteDirectory(subDirectory.FullName);
-            }
```

Comment:
This is the only bit I'm not clear on

Reply:
`GameInstance.TempDir` corresponds to `<GameRoot>/CKAN/temp`, but it has never been used for anything since I've been on the team. Since no code ever adds files or directories, there's no need to clear them out.

Reply:
Excellent, love code cleanup!